packages/mco/components/modals/app-manage-policies/manage-policy-view.tsx

```diff
@@ -82,8 +83,19 @@ const getAggregatedDRInfo = (
       assignedOn: formatTime(
         getLatestDate([acc.assignedOn, drpc?.metadata?.creationTimestamp])
       ),
+      volumeReplicationGroup:
```

Comment:
```suggestion
      isVolumeGroupReplicationEnabled:
```
For booleans, starting with is, has, uses, or supports makes the intent clearer.

Reply:
done :D